### DISCUSSION TITLE
How to use withouth disabling SIP

### DISCUSSION THREAD

**Discussion by tenmajkl** [asmvik/yabai#1045] [Q&A] [answered]
I'm a bit sceptical of disabling SIP, so I've decided to use yabai without it. So I've installed yabai, allowed accessibility settings but if i run brew services start yabai it doesn't work. Is it even possible to use yabai without disabling SIP?

**Comment by BR00l** [comment]
As @koekeishiya stated , yes, but it need to be build.
I hope for a 2 version release, one with SIP enabled, and one with disabled.
Im even ready to contribute with whatever I can for this feature.

**Comment by asmvik** [comment, marked as answer]
You can use the pre-built version without disabling SIP; just some features will not work (listed in the wiki). The same install instructions apply, just skip the ones that mention SIP and the scripting-addition.
